source/usr/lib/mcvirt/test/auth_tests.py

```diff
@@ -219,3 +219,45 @@ def test_delete_non_existant_superuser(self):
             self.parser.parse_arguments('permission --delete-superuser %s --global' %
                                         self.test_user,
                                         mcvirt_instance=self.mcvirt)
+
+    def test_change_password(self):
+        """Change the password of a user through the parser"""
+        new_password = 'new-password-here'
+        self.parser.parse_arguments('user change-password --target-user %s --new-password %s' %
+            (self.TEST_USERNAME, new_password)
+        )
+
+        # Try to create an RPC connection with the old credentials and check an AuthenticationError
+        # is raised
+        with self.assertRaises(AuthenticationError):
+            new_rpc = Connection(self.TEST_USERNAME, self.TEST_PASSWORD)
+
+        # Create a new RPC connection with the new credentials
+        try:
+            new_rpc = Connection(self.TEST_USERNAME, new_password)
```

Comment:
You could do an assertion here to ensure that this connection can do something, e.g. do a list or something?